lib/fog/compute/kubevirt/requests/get_offlinevm.rb

```diff
@@ -7,7 +7,7 @@ def get_offlinevm(name)
         end
 
         def get_raw_offlinevm(name)
-          object_to_hash( kubevirt_client.get_offline_virtual_machine(name, 'default') )
+          object_to_hash( kubevirt_client.get_offline_virtual_machine(name, @namespace) )
```

Comment:
no mock?

Reply:
We did not use mocks yet. We want to add them in the future.